homework_2/task_3.py

```diff
@@ -0,0 +1,19 @@
+number = int(input(" Enter number from 1 to 10: "))
+
+if number <= 1 or number > 10 :
+    print(" WRONG ")
+    exit(1)
+
+else :
+    if number % 2 == 0 and number % 3 == 0 :
```

Comment:
ზოგადად არარეკომენდირებულია ამხელა ლოგიკის else-ში ჩადება, მით უმეტეს ამ ამოცანაში, თორემ ხანდახან გვჭირდება ხოლმე. სწორია შენი უბრალოდ ეს იქნებოდა უკეთესი(გიტი მიშლის if-ებში ტაბებს):
number = int(input("Input natural number not exceeding 10: "))

if number > 10 or number < 1:
    print("Wrong input")
    exit(1)

if number % 2 == 0:
    print(2, end = ' ')
if number % 3 == 0:
    print(3, end=' ')
if number % 5 == 0:
    print(5, end=' ')
if number % 7 == 0:
    print(7, end=' ')